### ISSUE TITLE
Set styles for each event

### ISSUE THREAD

**Issue by HlukhovYe** [antoniandre/vue-cal-v4#526]
I need event.style property thet can add inline style to event cuz I get styles dynamic by API

**Comment by lehoaibaokg** [comment]
Yes. You can do that by adding more additional properties that store the styles to each event in the list and use the event slot to apply the custom styles.
Reference: https://antoniandre.github.io/vue-cal/#ex--custom-event-rendering
